qtopic/pyqtopics.py

```diff
@@ -72,46 +74,52 @@ def get_top_stories(topic):
             'title': title,
             'published': published
         }
-        return dict
+        return dict"""
```

Comment:
Why have you commented this method?

Reply:
I didn't find ```https://www.quora.com/topic/rss``` for any topic. That's why I have commented this method.